packages/iTwinUI-react/src/core/Select/Select.tsx

```diff
@@ -110,6 +117,15 @@ export type SelectProps<T> = {
    * @see [tippy.js props](https://atomiks.github.io/tippyjs/v6/all-props/)
    */
   popoverProps?: Omit<PopoverProps, 'onShow' | 'onHide' | 'disabled'>;
+  /**
+   * Enable multiple selection.
+   * @default false
+   */
+  multi?: boolean;
+  /**
+   * Custom renderer for the selected items in select.
+   */
+  selectedItemsRenderer?: (options: SelectOption<T>[]) => JSX.Element;
```

Comment:
You could try fancy TS where if `multi` is true, then `selectedItemRenderer` takes options list if false only single option

Reply:
no idea how to achieve now (at least now), but I will try something 😄

Reply:
Ok, I pushed something..
Guards are up for normal naming, but... it works, I think.. just not in storybook, cause for me it seems to be fully broken and cannot see any types at all (monorepo, I can bet)